### PR TITLE
Update README: Governance, Adopters and more links

### PR DESCRIPTION
**Pull request by oleg-nenashev**
This pull requests slightly improves the README page so that it provides more information about the project. This change also includes links to Code of Conduct, Governance and Adopters which are required for CDF Graduation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,13 +26,26 @@ Non-source downloads such as WAR files and several Linux packages can be found o
 Our latest and greatest source of Jenkins can be found on [GitHub]. Fork us!
 
 # Contributing to Jenkins
+
 Follow the [contributing guidelines](CONTRIBUTING.md) if you want to propose a change in the Jenkins core.
+For more information about participating in the community  contributing to the Jenkins project,
```
Comment (daniel-beck) [inline README.md:31]: Broken sentence?

File: README.md [hunk 2]
```diff
@@ -26,13 +26,26 @@ Non-source downloads such as WAR files and several Linux packages can be found o
...
+see [this page](https://www.jenkins.io/participate/).
 
-# Maintainers guidelines
 Documentation for Jenkins core maintainers is in the [maintainers guidelines](docs/MAINTAINERS.adoc).
 
 # News and Website
-All information about Jenkins can be found on our [website]. Follow us on Twitter [@jenkinsci].
+All information about Jenkins can be found on our [website].
```
Comment (daniel-beck) [inline README.md:37]: Other sections got an empty line between title and first sentence.

File: README.md [hunk 3]
```diff
@@ -26,13 +26,26 @@ Non-source downloads such as WAR files and several Linux packages can be found o
...
+Follow us on [Twitter](https://twitter.com/jenkinsci) or [LinkedIn](https://www.linkedin.com/company/jenkins-project/).
+
+# Governance
+
+See the [Jenkins Governance Document](https://www.jenkins.io/project/governance/) for information about the project's open governance, our philosophy and values, and the development practices.
```
Comment (daniel-beck) [inline README.md:42]:
```suggestion
See the [Jenkins Governance Document](https://www.jenkins.io/project/governance/) for information about the project's open governance, our philosophy and values, and development practices.
```

**Review by daniel-beck — CHANGES_REQUESTED** [review]
Requesting a change for the broken sentence, the rest looks good.

File: README.md (hunk 1 above)
Comment (oleg-nenashev) [reply README.md:31]:
```suggestion
For more information about participating in the community and contributing to the Jenkins project,
```

File: README.md (hunk 2 above)
Comment (oleg-nenashev) [reply README.md:37]:
```suggestion

All information about Jenkins can be found on our [website].
```

**Review by daniel-beck — APPROVED** [review]

**Review by markjacksonfishing — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -26,13 +26,27 @@ Non-source downloads such as WAR files and several Linux packages can be found o
...
+
+# Governance
+
+See the [Jenkins Governance Document](https://www.jenkins.io/project/governance/) for information about the project's open governance, our philosophy and values, and development practices.
+Jenkins Code of Conduct can be found [here](https://www.jenkins.io/project/conduct/).
+
+# Adopters
+
+Jenkins is used by millions of users and thousands of companies.
+See [this page] for the list of Jenkins adopters and their success stories.
```
Comment (timja) [inline README.md:49]:
```suggestion
See [adopters](https://www.jenkins.io/project/adopters/) for the list of Jenkins adopters and their success stories.
```

**Review by timja — CHANGES_REQUESTED** [review]

**Review by timja — APPROVED** [review]

**Comment by oleg-nenashev** [comment]
We may merge it in 24 hours if there is no negative feedback. Please see the [merge process documentation](https://github.com/jenkinsci/jenkins/blob/master/docs/MAINTAINERS.adoc#merge-process) for more information about the merge process

**Comment by oleg-nenashev** [comment]
Actually, maybe it makes sense to merge it right away, because it does not involve the production codebase. Opinions?

**Review by MarkEWaite — APPROVED** [review]
Looks great.  Thanks!  I think it is reasonable to merge immediately without waiting the 24 hours for merge objections.

**Comment by oleg-nenashev** [comment]
I will go ahead and merge it based on the feedback

**Pull request merged**